lib/common/client/middleware/response/json_parser.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Common
+  module Client
+    module Middleware
+      module Response
+        class JsonParser < Faraday::Response::Middleware
+          WHITESPACE_REGEX = /\A^\s*$\z/.freeze
+          MHV_SUCCESS_REGEX = /^success/i.freeze
```

Comment:
You're not communicating with MHV, right?

Reply:
Removing in https://github.com/department-of-veterans-affairs/gibct-data-service/pull/534